### ISSUE TITLE
<auto-generated> tag should not be translated

### ISSUE THREAD

**Issue by ErosGuil** [dotnet/wcf#3159]
When generating a service reference, on a french computer, here is what is generated:

```
//------------------------------------------------------------------------------
// <généré automatiquement>
//     Ce code a été généré par un outil.
//
//     Les changements apportés à ce fichier peuvent provoquer un comportement incorrect et seront perdus si
//     le code est regénéré.
// </généré automatiquement>
//------------------------------------------------------------------------------
```

`<généré automatiquement>` should be kept as `<auto-generated>`

**Comment by mlacouture** [comment]
Hi @ErosGuil 
thanks for reporting this issue.  I'll pass it to our localizers.

**Comment by mlacouture** [comment]
BTW: what file is this? is it a resx designer file? if so, this needs to be moved to the dotnet/corefx or CoreCLR repo.

**Comment by jeremyVignelles** [comment]
Hi @mlacouture , it's the header of the generated Reference.cs file that was generated by dotnet-svcutil.

One side note : do you feel like the content of the comment should be translated?

I know visual studio has always written those kind of comments in the user's language, but it's really annoying for open source project with developpers with different languages, when they regenerate files (like resx).

What do you feel about it?

**Comment by mlacouture** [comment]
@jeremyVignelles, this seems to me by design, this comes from the CodeDom API and it seems it has been like this forever, I don't think we will be able to make any changes in this area.  

yeah, it may be annoying in your scenario, the original intent is that people reading this code can understand it in their own language.

thank you for providing your feedback.

**Comment by sharwell** [comment]
@mlacouture This tag is used for identify generated code for Roslyn analyzers. It is required to appear as one of the following:

* `<auto-generated>`
* `<auto-generated/>`
* `<autogenerated>`
* `<autogenerated/>`

Translations are not allowed, and attempting to do so will break compiler functionality for international users.

**Comment by mlacouture** [comment]
I see, reopened.

**Comment by dasetser** [comment]
This has been fixed in dotnet-svcutil 1.0.4.
